src/components/Header.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Link, useLocation } from 'react-router-dom';
```

Comment:
- Optional
I would recommend using `NavLink` instead of `Link`. You can read about the differences between the two [here](https://www.educative.io/answers/what-are-the-differences-between-navlink-link-and-a-in-react).